app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -1,25 +1,27 @@
 # noinspection RubyArgCount
 class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::Toolbar::Basic
-  button_group('middleware_server_vmdb', [
+  button_group('middleware_server_monitoring', [
     select(
-      :middleware_server_vmdb_choice,
-      'fa fa-cog fa-lg',
-      t = N_('Configuration'),
+      :middleware_server_monitoring_choice,
+      'product product-monitoring fa-lg',
+      t = N_('Monitoring'),
```

Comment:
I think that at Middleware Server detail level, the configuration button was disabled and there was not plan to manage it.
I remove it with the motivation to not leave a broken button in the provider.
My thought was that if we need to add "Configuration" option we can enable later once we have the backend ready.